### PR TITLE
feat-input

### PR DESCRIPTION
**Pull request by iamsaurav28**
https://user-images.githubusercontent.com/90086339/153057136-b5c221b3-b53b-4151-9670-fffde4dd5a49.mp4

### CODE REVIEW

File: Comp-layouts/input.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (Sameony) [inline Comp-layouts/input.html]: add a clear and relevant document title here

File: Stylesheets/components/input.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  color: white;
+  border-radius: 0.2rem;
+  display: flex;
+  flex-direction: column;
+}
+
+.input {
+  min-width: 20rem;
+  padding: 0.1rem;
+  border-radius: 0.1rem;
```
Comment (Sameony) [inline Stylesheets/components/input.css:17]: for very small values, you can use pixels instead.
values such as 0.5rem and so on are okay, but 0.1rem is basically saying 1-2px. not much scope for responsiveness in such a small value.

File: Stylesheets/components/input.css [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@import url("/stylesheets/main.css");
+
+.input-holder {
+  width: fit-content;
+  margin: 1rem;
+  padding: 0.5rem;
+  background-color: var(--bg-black);
+  color: white;
```
Comment (Sameony) [inline Stylesheets/components/input.css]: use the variable of white color you created here for consistency

**Review by Sameony — COMMENTED** [review]
do try to maintain consistency throughout the code

File: Stylesheets/components/input.css (hunk 2 above)
Comment (iamsaurav28) [reply Stylesheets/components/input.css:17]: yes you  right but some are saying dont change units in between so i have kept everything in rem only

File: Comp-layouts/input.html (hunk 1 above)
Comment (iamsaurav28) [reply Comp-layouts/input.html]: okay i have named it

File: Stylesheets/components/input.css (hunk 3 above)
Comment (iamsaurav28) [reply Stylesheets/components/input.css]: oh yes right i did changes

**Review by Gautam2305 — COMMENTED** [review]
looks clean. good variations with the input styles. Looks good to merge

**Review by Gautam2305 — APPROVED** [review]

**Review by Sameony — APPROVED** [review]
You can merge it now. Good work.

**Pull request merged**